tests/components/samsungtv/test_config_flow.py

```diff
@@ -334,22 +441,32 @@ def remote_fixture():
     assert entry.data[CONF_ID] == "fake_uuid"
 
 
-async def test_autodetect_websocket(hass, remote):
+async def test_autodetect_websocket(hass, remote, remotews):
     """Test for send key with autodetection of protocol."""
-    with patch("homeassistant.components.samsungtv.config_flow.Remote") as remote:
+    with patch(
+        "homeassistant.components.samsungtv.bridge.Remote", side_effect=OSError("Boom"),
+    ), patch("homeassistant.components.samsungtv.bridge.SamsungTVWS") as remotews:
         result = await hass.config_entries.flow.async_init(
             DOMAIN, context={"source": "user"}, data=MOCK_USER_DATA
         )
         assert result["type"] == "create_entry"
         assert result["data"][CONF_METHOD] == "websocket"
-        assert remote.call_count == 1
-        assert remote.call_args_list == [call(AUTODETECT_WEBSOCKET)]
+        assert remotews.call_count == 1
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5152307)